tests/integration/service_test.py

```diff
@@ -102,6 +102,12 @@ def test_create_container_with_cpu_quota(self):
         container.start()
         self.assertEqual(container.get('HostConfig.CpuQuota'), 40000)
 
+    def test_create_container_with_shm_size(self):
+        service = self.create_service('db', shm_size=67108864)
```

Comment:
Isn't this the default value? `shm_size` doesn't seem to work - the following configuration still returns `"ShmSize": 67108864`:

```
  db:
    image: postgres:11.1
    shm_size: '2gb'
```

Ditto for `shm_size: 2147483648`.

Reply:
[Others](https://stackoverflow.com/q/55416904/96588) seem to have the same problem.